snuba/utils/streams/batching.py

```diff
@@ -1,16 +1,34 @@
 import logging
 import time
 from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import (
+    Any,
+    MutableMapping,
+    MutableSequence,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+)
 
 from confluent_kafka import (
-    Consumer, KafkaError, KafkaException,
-    OFFSET_BEGINNING, OFFSET_END, OFFSET_STORED, OFFSET_INVALID
+    Consumer,
+    KafkaError,
+    KafkaException,
+    Message,
+    OFFSET_BEGINNING,
+    OFFSET_END,
+    OFFSET_STORED,
```

Comment:
nit: offset_stored and offset_invalid are in the wrong order

Reply:
Thanks for noticing, I made this case-sensitive lex sorted with 5220887b9cd2ffcb60a8b529a2407d1c115beb22.

Reply:
Actually, I guess the vscode "Sort Imports" functionality for Python isn't a true lex sort, it sorts all-uppercase items first, which is OK too, I guess.